src/renderer/services/bitcoin/common.ts

```diff
@@ -30,7 +30,7 @@ const testnetHaskoinProvider = new HaskoinProvider(
 )
 
 const LOWER_FEE_BOUND = 1
-const UPPER_FEE_BOUND = 900
+const UPPER_FEE_BOUND = 2000
```

Comment:
Why not make this double `UPPER_FEE_BOUND` in `xchainjs/xchain-bitcoin`?

Reply:
not necessary